python/operations.cpp

```diff
@@ -72,6 +73,15 @@ template <typename T> void bind_sort_dim(py::module &m) {
       doc.c_str());
 }
 
+template <typename T> void bind_sort_inner(py::module &m) {
```

Comment:
```suggestion
template <typename T> void bind_sort_variable(py::module &m) {
```
It is irrelevant here that this *currently* only supports sorting the inner dim. If we add support later, the name here would be wrong.